mmv1/templates/terraform/pre_update/org_policy_policy.go.tmpl

```diff
@@ -0,0 +1,30 @@
+<%- # the license inside this block applies to this file
+	# Copyright 2024 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+updateMask := []string{}
+
+if d.HasChange("spec") {
```

Comment:
Does adding update_mask: true in the YAML file not handle this correctly?

Reply:
The default update mask doesn't add `policy.` before each field name as the API requires.